app/controllers/katello/api/v2/subscriptions_controller.rb

```diff
@@ -241,6 +254,14 @@ def index_organization
       return subscriptions
     end
 
+    def consumed_subscriptions(consumer, organization = current_organization)
+      entitlements = consumer.consumed_entitlements
+      entitlements.delete_if do |entitlement|
+        pool = Pool.find_pool(entitlement.poolId)
+        !pool.provider?(organization)
+      end
```

Comment:
The whole function
On Sep 11, 2015 4:03 PM, "John Mitsch" <notifications@github.com> wrote:

> In app/controllers/katello/api/v2/subscriptions_controller.rb
> <https://github.com/Katello/katello/pull/5446#discussion_r39311042>:
>
> > @@ -241,6 +254,14 @@ def index_organization
> >        return subscriptions
> >      end
> >
> > +    def consumed_subscriptions(consumer, organization = current_organization)
> > +      entitlements = consumer.consumed_entitlements
> > +      entitlements.delete_if do |entitlement|
> > +        pool = Pool.find_pool(entitlement.poolId)
> > +        !pool.provider?(organization)
> > +      end
>
> do you mean in the delete_if block or rewrite the method?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Katello/katello/pull/5446/files#r39311042>.
>